### ISSUE TITLE
Some question about the implemention

### ISSUE THREAD

**Issue by chenhch8** [ZhitingHu/logicnn#4]
Hi!
Thanks for your release of the code!
I have some question here. It seems that the implemention is different from the formula(5). Besides, I don't know what's the definition of the 'r' function in formula(4). Could you tell me what it is?
What's more, I am also curiosty about the implemention about NER and it is difficult to understand it if just read your paper. Would you mind release your code about NER?
Thanks very mush!

**Comment by martiansideofthemoon** [comment]
I agree with this significant difference in the implementation. We have recently published a paper in EMNLP 2018 about the reproducibility of the sentiment analysis experiments in this paper (https://arxiv.org/abs/1808.07733) and released a codebase for it (http://github.com/martiansideofthemoon/logic-rules-sentiment)
